lib/commands/reshim.bash

```diff
@@ -88,6 +88,14 @@ write_shim_script() {
   local shim_path
   shim_path="$(asdf_data_dir)/shims/$executable_name"
 
+  # Temporary fix for #1231
+  # shellcheck disable=SC2016
+  if ! grep '^exec \${ASDF_DIR}' "$shim_path" >/dev/null; then
+    # shellcheck disable=SC2016
+    sed -i.bak -e 's/^exec .*\/bin\/asdf /exec ${ASDF_DIR}\/bin\/asdf /' "$shim_path"
+    rm -rf "$shim_path".bak
+  fi
```

Comment:
If we do end up merging this I suggest we remove this `if` block and instead make sure there is an easy way for developers to delete all their shims and re-generate them from scratch.